elan/rpc/eclient.go

```diff
@@ -108,9 +108,16 @@ func (e *elanClient) uploadOne(entry *uploadinfo.Entry) error {
 	if err != nil {
 		return err
 	}
-	defer wr.Close()
+
+	defer func() {
+		if closeErr := wr.Close(); err == nil {
+			err = closeErr
+		}
+	}()
```

Comment:
Should we do this for f.Close() above as well?

Reply:
I think we care about the write err more but it wont hurt to add it to f.Close() also